### DISCUSSION TITLE
Keep Speed and Mouse Wheel

### DISCUSSION THREAD

**Discussion by Mogli12** [Mogli12/KeyboardSteer#769] [Announcements]
Starting with build 38 it is possible to change the keep speed value with the mouse wheel. 

The idea of keep speed is to simulate the joystick like in a Fendt. You can dis-/enable it with Ctrl+Del. You can accelerate and brake with the normal controls. Additionally, you can now change the speed with the mouse wheel. Unfortunately, this is the default action binding of camera zoom of the outside camera.

What is your experience with keep speed?

**Comment by Kynuska** [comment]
Hi.
I toyed with the same idea for binding to adjust the cruise control speed. 99% I play inside the cabin.

It only works for me in one direction. (build 38) Deceleration. The wheel up does not even respond to zooming out. 
Kynuska
